src/core/mac/mac.cpp

```diff
@@ -748,28 +735,21 @@ ThreadError Mac::ProcessReceiveSecurity(const Address &aSrcAddr, Neighbor *aNeig
     return error;
 }
 
-extern "C" void otPlatRadioSignalReceiveDone(void)
-{
-    sReceiveDoneTask.Post();
-}
-
-void Mac::ReceiveDoneTask(void *aContext)
+extern "C" void otPlatRadioReceiveDone(ThreadError error)
 {
-    sMac->ReceiveDoneTask();
+    sMac->ReceiveDoneTask(error);
 }
 
-void Mac::ReceiveDoneTask(void)
+void Mac::ReceiveDoneTask(ThreadError aError)
```

Comment:
Here you have aError which I think more closely matches the style throughout.  Just above you have "error".